src/pages/ServiceDetails/ServiceTraces.tsx

```diff
@@ -246,78 +259,93 @@ class ServiceTracesC extends React.Component<ServiceTracesProps, ServiceTracesSt
             <Card>
               <CardBody>
                 <RenderHeader>
-                  <Grid>
-                    <GridItem span={2}>
-                      <Text component={TextVariants.h5} style={{ display: '-webkit-inline-box', marginRight: '10px' }}>
-                        Interval Trace
-                      </Text>
-                      <ToolbarDropdown
-                        options={this.state.traceIntervalDuration}
-                        value={this.state.traceIntervalDuration[this.state.selectedTraceIntervalDuration]}
-                        handleSelect={key => this.handleIntervalDuration(key)}
-                      />
-                    </GridItem>
-                    <GridItem span={2}>
-                      <Text component={TextVariants.h5} style={{ display: '-webkit-inline-box', marginRight: '10px' }}>
-                        Limit Results
-                      </Text>
-                      <ToolbarDropdown
-                        options={config.tracing.configuration.limitResults}
-                        value={config.tracing.configuration.limitResults[this.state.selectedLimitSpans]}
-                        handleSelect={key => this.handleLimitDuration(key)}
-                      />
-                    </GridItem>
-                    <GridItem span={2}>
-                      <Text
-                        component={TextVariants.h5}
-                        style={{ display: '-webkit-inline-box', marginLeft: '-40px', marginRight: '10px' }}
-                      >
-                        Status Code
-                      </Text>
-                      <ToolbarDropdown
-                        options={config.tracing.configuration.statusCode}
-                        value={config.tracing.configuration.statusCode[this.state.selectedStatusCode]}
-                        handleSelect={key => this.handleStatusCode(key)}
-                      />
-                    </GridItem>
-                    <GridItem span={2}>
-                      <div style={{ marginTop: '10px' }}>
-                        <Checkbox
-                          label="Adjust time"
-                          isChecked={this.state.fixedTime}
-                          onChange={checked => {
-                            this.setState({ fixedTime: checked });
-                          }}
-                          aria-label="adjust-time-chart"
-                          id="check-adjust-time"
-                          name="check-adjust-time"
+                  <Toolbar>
```

Comment:
The Grid -> Toolbar was a good change especially for accessibility.

Reply:
Yeah the point is that I not used toolbar because when I tried there was so issues in scale it seems that they fixed it. Thanks @mtho11